### DISCUSSION TITLE
Search Function Limitation in Older Transactions

### DISCUSSION THREAD

**Discussion by BobWs** [victorbalssa/abacus#325] [General]
Hi,

I’ve noticed that when using the search function to find an older transaction, it only returns results from the currently active/selected month. Is this behavior intentional or a potential bug?

It would be really helpful if the search function could search across all transactions, not just those within the selected month. Is it possible to modify or extend the search functionality to include all transactions?

Thank you!

**Comment by victorbalssa** [comment]
Hello, I will track it with https://github.com/victorbalssa/abacus/issues/294 👍🏻
